Example/test-suite/tests/Colors.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+import { processColor } from 'react-native-reanimated';
+
+export const name = 'Colors';
+
+export function test(t) {
+  t.describe('colors', () => {
+    // test basic colors
+    t.it('process red color', () => {
+      const padding = 2;
+      // test red
+      const redStr = processColor('red')
+        .toString(16)
+        .substring(padding + 0, padding + 2);
+      const redNum = Number.parseInt(redStr, 16);
+      t.expect(redNum).not.toBe(0);
```

Comment:
Can we check exactly if it's the correct value?

Reply:
fixed 7a5999d